.github/workflows/camunda-platform-release.yml

```diff
@@ -173,14 +173,15 @@ jobs:
           # see https://maven.apache.org/maven-release/maven-release-plugin/perform-mojo.html#workingDirectory
           export ZBCTL_ROOT_DIR=${PWD}
 
-          # -P-integration-tests,-autoFormat
+          # -DskipQaBuild -P-autoFormat
           #   In order to disable QA/IT and the auto formatting during the release.
           #   It is unnecessary to release integration tests, and we don't want to run the auto-formatting unnecessarily.
           ./mvnw release:perform -B \
             -Dgpg.passphrase="${{ steps.secrets.outputs.MAVEN_CENTRAL_GPG_SIGNING_KEY_PASSPHRASE }}" \
             -DlocalCheckout=${{ inputs.dryRun }} \
-            -P-integration-tests,-autoFormat \
-            -Darguments='-T0.5C -P-integration-tests,-autoFormat -DskipChecks=true -DskipTests=true -Dspotless.apply.skip=false -Dskip.central.release=${SKIP_REPO_DEPLOY} -Dskip.camunda.release=${SKIP_REPO_DEPLOY} -Dzbctl.force -Dzbctl.rootDir=${ZBCTL_ROOT_DIR} -Dgpg.passphrase="${{ steps.secrets.outputs.MAVEN_CENTRAL_GPG_SIGNING_KEY_PASSPHRASE }}"'
+            -DskipQaBuild \
+            -P-autoFormat \
+            -Darguments='-T0.5C -P-autoFormat -DskipChecks=true -DskipTests=true -Dspotless.apply.skip=false -Dskip.central.release=${SKIP_REPO_DEPLOY} -Dskip.camunda.release=${SKIP_REPO_DEPLOY} -Dzbctl.force -Dzbctl.rootDir=${ZBCTL_ROOT_DIR} -Dgpg.passphrase="${{ steps.secrets.outputs.MAVEN_CENTRAL_GPG_SIGNING_KEY_PASSPHRASE }}"'
```

Comment:
Do we have to add the `-DskipQaBuild` here inside of the `-Darguments` string as well?

Reply:
Yes we do. That is the part which is executed as part of the deploy during release:perform

Reply:
This is the same PR as before, I just had to remake it. Getting all sorts of CI issues after rebasing and it seemed simpler to just remake the PR and make sure I didn't mess up anything

Reply:
Oh I see what you're saying, I need to move -DskipQaBuild inside the -Darguments, I'll update that now

Reply:
I also made a small tweak to make it more consistent with other environment variables so it's clearer for the user.

If the property is not present: QA build occurs (profile is active by default basically)
If the property is present with a value of 'true': QA build occurs (-DskipQaBuild=true)
If the property is present with a value of 'false': QA build does not occur (-DskipQaBuild=false)

This will follow the same behavior as other variables like skipTests.

Reply:
> If the property is not present: QA build occurs (profile is active by default basically)
If the property is present with a value of 'true': QA build occurs (-DskipQaBuild=true)
If the property is present with a value of 'false': QA build does not occur (-DskipQaBuild=false)

Just to make sure I am not going mad, shouldn't this be:
If the property is not present: QA build occurs (profile is active by default basically)
If the property is present with **any other value than** 'true': QA build occurs (-DskipQaBuild=**false**)
If the property is present with a value of **'true'**: QA build does not occur (-DskipQaBuild=**true**)

Reply:
You are correct, this is what happens when I try to write up explanations when I'm doing too many things at once. I just tested to make sure.

No property, qa build.
Property false, qa build.
Property true, qa build

Sorry for the confusion and churn on this PR.